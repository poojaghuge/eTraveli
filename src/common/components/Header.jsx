import * as React from "react";
import { styled, alpha } from "@mui/material/styles";
import AppBar from "@mui/material/AppBar";
import Box from "@mui/material/Box";
import Toolbar from "@mui/material/Toolbar";

export default function Header() {
  return (
    <Box sx={{ flexGrow: 1 }}>
      <AppBar position="static" style={{backgroundColor:'#f0f0f021'}}>
        <Toolbar>
          <div>Movie App</div>
        </Toolbar>
      </AppBar>
    </Box>
  );
}
