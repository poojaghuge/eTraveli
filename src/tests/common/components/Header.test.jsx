import { render, screen } from "@testing-library/react";
import React from "react";
import renderer from "react-test-renderer";
import Header from "../../../common/components/Header";

describe("Test Header", () => {
  it("test if matches the DOM Snapshot", () => {
    const domTree = renderer.create(<Header />).toJSON();
    expect(domTree).toMatchSnapshot();
  });

    it("test if selected prop is not passed default div should render", () => {
    const {getByText} = render(<Header />);
    const element = getByText('Movie App');
    expect(element).toBeInTheDocument();
  });

});
