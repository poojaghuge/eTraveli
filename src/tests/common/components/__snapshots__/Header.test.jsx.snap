// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header test if matches the DOM Snapshot 1`] = `
<div
  className="MuiBox-root css-i9gxme"
>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
    style={
      Object {
        "backgroundColor": "#f0f0f021",
      }
    }
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-hyum1k-MuiToolbar-root"
    >
      <div>
        Movie App
      </div>
    </div>
  </header>
</div>
`;
